packages/vector_graphics/test/vector_graphics_test.dart

```diff
@@ -502,39 +502,6 @@ void main() {
     expect(exception.toString(), contains(loader.toString()));
   });
 
-  testWidgets(
```

Comment:
This test doens't pass anymore, and seems to be depending on internals of the web implementation so I deleted it.